server/methods/setLastVisitedRoom.js

```diff
@@ -0,0 +1,6 @@
+Meteor.methods({
+	setLastVisitedRoom(userId, room) {
+		Meteor.users.update(userId, { $set: { lastVisitedRoom: room } });
```

Comment:
Do not update any data based on user id from parameter, that way I'm able to change any user.

Use `this.userId` instead and check if `room` is a string before proceed.